mysql/README.md

```diff
@@ -137,7 +137,7 @@ datasets:
     params:
       mysql_host: localhost
       mysql_tcp_port: 3306
-      mysql_db: spice_test
+      mysql_db: spice_demo
```

Comment:
```suggestion
      mysql_db: mysql
```
mysql databases have a default name 'mysql' - this was an error I encountered with. I don't think we should use databse name/table name from the demo as this is different.